### PR TITLE
[BOUNTY] Hecata Bloodsucker Clan

### PR DESCRIPTION
**Pull request by Marmio64**
# Document the changes in your pull request

Adds the Hecata clan as a bloodsucker clan.

Link to bounty: https://forums.yogstation.net/threads/15-add-a-new-bloodsucker-clan-ss13-server.26337/

Upsides-
Necromancy: The prime ability of the clan. Hefty blood cost, but lets you quickly convert the dead into temporary zombie vassals. They last 5 minutes at level 1, and up to 15 minutes at level 4 ability level. It can also be used on dead vassals you already own in order to resurrect them permanently. Either way, the victim becomes a Sanguine Zombie, which is like a mix between a non infectious zombie and a infectious one. It can't use guns, and has some slowdown, takes slightly more damage, has soft crit resistance, and high punch damage. Note: Can't be used to farm blood due to Sanguine Zombies not having any. Also doesn't work on Monster Hunters or other Bloodsuckers.

Spirit Call: Summons temporary wraiths which attack nearby non dead. They will attack your vassals too if they aren't zombified! They are weak though and primarily good as distractions for escapes or bolstering your horde...

Deathly Communion: On a cooldown you can send messages to all your vassals, allowing you to coordinate your zombie horde.

Favorite vassal ability: Spirit Call. Favorite vassal is also turned into a Sanguine Zombie if they weren't one already.

Downsides-
Painful Bite: Feeding is painful enough to the victim that the bloodsucker is incapable of stealthily feeding on people, meaning it has to be done while aggressively grabbing them.
No Blood Altar: Using the altar is no longer of interest to you: Thus you can't do tasks for it and level up via it.

Unique Clan Goal: Necromance X amount of people.

Overall, Hecata focuses on a louder approach and on their vassals, similar to the Tzimisce clan. However, rather than a focus on having a few high power vassals, they make hordes of disposable ones. 

# Spriting
hecatapowers
(Unused skeleton sprite deleted, haven't gotten to updating this on the PR description)

# Wiki Documentation

Hecata

The unified clan of death. The Hecata utilize the art of Oblivion to raise the dead, summon hostile ghosts, and communicate with their vassals from afar.

Joining the clan has the following side effects
-You gain 3 abilities.
--Necromancy: Raise dead people as temporary zombie vassals loyal to you, or revive existing vassals permanently.
--Spirit Call: Summon angry wraiths which attack nearby people who aren't zombies (includes non zombified vassals!)
--Deathly Communion: Sends a message to all your vassals no matter their location.

-You can no longer silently feed. Feeding must be done with an aggressive grab.
-You can't interact with blood altars to do tasks or gain ranks.

# Changelog

:cl:  
rscadd: Adds the clan of death to the bloodsuckers, The Hecata Clan! Avid necromancers whose feeding is too painful to be stealthy.
/:cl:

### CODE REVIEW

File: code/game/mecha/equipment/weapons/melee_weapons.dm [hunk 1]
```diff
@@ -247,20 +272,266 @@
...
+	energy_drain = 40
+	weapon_damage = 25			//Very smashy
+	precise_weapon_damage = 30
+	attack_speed_modifier = 2.5	//Very slow
+	fauna_damage_bonus = 40
+	structure_damage_mult = 4	//Good for stationary objects
+	attack_sharpness = SHARP_NONE
+	sword_wound_bonus = 20		//Makes bones sad :(
+	mob_strike_sound = 'sound/effects/meteorimpact.ogg'	//WHAM
+	hit_effect = ATTACK_EFFECT_SMASH					//POW
```
Comment (tattax) [inline code/game/mecha/equipment/weapons/melee_weapons.dm]: promising stuff, hope oyu can add more

Comment (Marmio64) [reply code/game/mecha/equipment/weapons/melee_weapons.dm]: This is a real old branch, I premade most of this in advance. As of now I'm deciding whether or not to just make a whole new branch and transfer my stuff or fix this one up.

**Comment by tattax** [comment]
this might be set on holding depending on how https://github.com/yogstation13/Yogstation/pull/18514 goes btw

**Comment by JohnFulpWillard** [comment]
good pr

**Comment by Marmio64** [comment]
> this might be set on holding depending on how #18514 goes btw

How long do you expect that pr to take to finish? I could probably get this one tied up and ready in a couple days but I may wait if your PR would be ready first as it may save me some trouble. Asides from the merge conflicts, I just need to try to verify deathly communion works, fix the blood cost of necromancy, and fix wraiths not disappearing on death.

**Comment by Marmio64** [comment]
there's still some stuff to work on and fix (biggest one would be Necromancy using 70 blood instead of the 35 blood it is supposed to use) but this is in a decent state and would be ready for test merging so balancing can be addressed.

**Comment by tattax** [comment]
> > this might be set on holding depending on how #18514 goes btw
> 
> How long do you expect that pr to take to finish? I could probably get this one tied up and ready in a couple days but I may wait if your PR would be ready first as it may save me some trouble. Asides from the merge conflicts, I just need to try to verify deathly communion works, fix the blood cost of necromancy, and fix wraiths not disappearing on death.

i finish it today or tomorrow

**Review by cuackles — CHANGES_REQUESTED** [review]
- wraith side sprites need a lil tweek, trow the eyes like 1 tile more to the front so it actually does look like 1 eyes instead of 1 in each side 
- there is an icon without a proper name, please give it one

**Comment by Marmio64** [comment]
> * wraith side sprites need a lil tweek, trow the eyes like 1 tile more to the front so it actually does look like 1 eyes instead of 1 in each side
> * there is an icon without a proper name, please give it one

done

**Comment by Marmio64** [comment]
i dont know what i did wrong in this branch but any time a new PR is merged into the upstream the html/changelog.html file gets conflicts until i merge upstream into my branch.

**Comment by Marmio64** [comment]
Due to the above comment it may be a while in-between fixing it until this is going to be testmerged/merged to avoid bloat.

**Review by cuackles — APPROVED** [review]

**Review by Moltijoe — APPROVED** [review]

**Comment by Marmio64** [comment]
Hopefully fixed and moved to a new pr:#18971

**Pull request closed without merging**